src/set_theory/game/pgame.lean

```diff
@@ -1089,54 +1120,53 @@ theorem lt_iff_sub_pos {x y : pgame} : x < y ↔ 0 < y - x :=
      ... ≤ y : (add_zero_relabelling y).le⟩
 
 /-- The pre-game `star`, which is fuzzy/confused with zero. -/
-def star : pgame := pgame.of_lists [0] [0]
+def star : pgame.{u} := of_lists [0] [0]
 
-instance inhabited_star_left_moves : inhabited star.left_moves :=
-show (inhabited (fin 1)), by apply_instance
+@[simp] theorem star_left_moves : star.left_moves = ulift (fin 1) := rfl
+@[simp] theorem star_right_moves : star.right_moves = ulift (fin 1) := rfl
 
-instance inhabited_star_right_moves : inhabited star.right_moves :=
-show (inhabited (fin 1)), by apply_instance
+@[simp] theorem star_move_left (x) : star.move_left x = 0 :=
+show (of_lists _ _).move_left x = 0, by simp
+@[simp] theorem star_move_right (x) : star.move_right x = 0 :=
+show (of_lists _ _).move_right x = 0, by simp
 
-theorem star_lt_zero : star < 0 :=
-by rw lt_def; exact
-or.inr ⟨⟨0, zero_lt_one⟩, (by split; rintros ⟨⟩)⟩
+instance unique_star_left_moves : unique star.left_moves :=
+@equiv.unique _ (fin 1) _ equiv.ulift
+instance unique_star_right_moves : unique star.right_moves :=
+@equiv.unique _ (fin 1) _ equiv.ulift
 
+theorem star_lt_zero : star < 0 :=
+by { rw lt_zero, use default, rintros ⟨⟩ }
 theorem zero_lt_star : 0 < star :=
-by rw lt_def; exact
-or.inl ⟨⟨0, zero_lt_one⟩, (by split; rintros ⟨⟩)⟩
+by { rw zero_lt, use default, rintros ⟨⟩ }
 
 /-- The pre-game `ω`. (In fact all ordinals have game and surreal representatives.) -/
 def omega : pgame := ⟨ulift ℕ, pempty, λ n, ↑n.1, pempty.elim⟩
 
 theorem zero_lt_one : (0 : pgame) < 1 :=
-begin
-  rw lt_def,
-  left,
-  use ⟨punit.star, by split; rintro ⟨ ⟩⟩,
-end
+by { rw zero_lt, use default, rintro ⟨⟩ }
 
 /-- The pre-game `half` is defined as `{0 | 1}`. -/
 def half : pgame := ⟨punit, punit, 0, 1⟩
 
-@[simp] lemma half_move_left : half.move_left punit.star = 0 := rfl
+@[simp] theorem half_left_moves : half.left_moves = punit := rfl
+@[simp] theorem half_right_moves : half.right_moves = punit := rfl
+@[simp] lemma half_move_left (x) : half.move_left x = 0 := rfl
+@[simp] lemma half_move_right (x) : half.move_right x = 1 := rfl
```

Comment:
I think it would be worth trying to train `simps` to generate these

Reply:
How does `simps` work?

Reply:
If `pgame` were a structure, rather than an inductive type, we would use `initialize_simps_projections`.

However as it is an inductive type I don't think we can do anything.

Reply:
I've raised this on zulip. https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/initialize_simps_projections.20for.20inductive.20types